### ISSUE TITLE
Superchats don't seem to stick to the top anymore

### ISSUE THREAD

**Issue by cohlexyz** [jaw-sh/stream-nexus#24]
I know that the duration depends on the amount donated so sometimes they go right through if the chat is slow but $100 definitely used to stick around for quite a while. In the Kick VOD from Tuesday at around 2:10:30:

https://github.com/jaw-sh/stream-nexus/assets/142505474/0b1e04f0-3b5f-4605-9b54-b883df0f9825

**Comment by jaw-sh** [comment]
Oh I changed the way the chat layout is set up recently, I must have broken it.

**Comment by ghost** [comment]
I can't seem to reproduce this in testing.

**Comment by cohlexyz** [comment]
I assume the change that broke it hasn't been pushed yet. I haven't tested if it happens on the master branch, just noticed it on stream

**Comment by cohlexyz** [comment]
I just checked a7bc9de and it seems to me like the issue was fixed but only for normal browsers. The embedded browser in OBS can sometimes behave a bit weird. As it turns out this line:
https://github.com/jaw-sh/stream-nexus/blob/a7bc9de241bf7e5e0c06e37edf85fbec51e3a33e/public/script.js#L146
Works fine in Firefox or Chrome but in CEF (not sure if it's version dependent) `document.body.scrollHeight` is just zero. To debug these kinds of issues remote debugging is useful which can be enabled by launching obs with `--remote-debugging-port=9222` and then opening Chromium at `chrome://inspect/#devices`. For comparison (remote debugging in the background, dev tools for normal chrome in foreground):

![Screenshot_20240518_101714](https://github.com/jaw-sh/stream-nexus/assets/142505474/acb68dfd-85a8-41f3-a23f-dad208d458ad)


Changing the line to
```js
let space = window.innerHeight / 2;
```
seems to fix the issue for me.

Also very minor thing I noticed when comparing my local overlay to yours on stream is that it seems yours doesn't use the bold version of the font.
